pkgs/applications/networking/ngadmin/default.nix

```
@@ -0,0 +1,22 @@
+{ stdenv, fetchgit, autoreconfHook, readline }:
+
+stdenv.mkDerivation {
+  pname = "ngadmin";
+  version = "unstable";
```

```suggestion
  version = "unstable-XXXX-XX-XX";
```

Sadly I cannot visit the git to check the date of the commit